bentoml/_internal/configuration/__init__.py

```diff
@@ -28,14 +26,29 @@ class version_mod:
 CONFIG_ENV_VAR = "BENTOML_CONFIG"
 
 
-def expand_env_var(env_var: t.Union[str, bytes]) -> str:
+@t.overload
+def expand_env_var(env_var: None) -> None:
+    ...
+
+
+@t.overload
+def expand_env_var(env_var: str) -> str:
+    ...
+
+
+@t.overload
+def expand_env_var(env_var: t.Union[str, bytes]) -> t.Union[str, bytes]:
+    ...
+
+
+def expand_env_var(
+    env_var: t.Optional[t.Union[str, bytes]]
```

Comment:
let's change change this to `str -> str` ?  there's only one usage of this function